src/rebar_plugins.erl

```diff
@@ -105,7 +105,7 @@ handle_plugin(Profile, Plugin, State, Upgrade) ->
         %% Add newly built deps and plugin to code path
         State3 = rebar_state:update_all_plugin_deps(State2, Apps),
         NewCodePaths = [rebar_app_info:ebin_dir(A) || A <- ToBuild],
-        code:add_pathsa(CodePaths),
+        code:add_pathsa(NewCodePaths),
```

Comment:
Should it be `NewCodePaths` or `CodePaths++NewCodePaths` considering the compiler may have dropped paths on 103?